props/prop_base.h

```diff
@@ -551,6 +551,19 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     return current_config->ohm < NO_BLADE;
   }
 
+  virtual void SpeakBladeID(float id) {
+// Props should define this to use sound library.
+#ifndef USE_SOUND_LIBRARY_FOR_SPEAK_BLADE_ID
```

Comment:
prop_base.h doesn't need to know about USE_SOUND_LIBRARY_FOR_BLADE_ID

Reply:
I need to stop working till 4am.
Fixed.